### PR TITLE
add: `highlight-pointer-bin`

### PR DESCRIPTION
**Pull request by 0x262D**

### CODE REVIEW

File: packages/highlight-pointer-bin/highlight-pointer-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,27 @@
+maintainer="0x262D <pacstall@copypaste.anonaddy.me>"
+
```
Comment (Henryws) [inline packages/highlight-pointer-bin/highlight-pointer-bin.pacscript:2]:
```suggestion
```

Comment (Henryws) [inline packages/highlight-pointer-bin/highlight-pointer-bin.pacscript:2]:
```suggestion
```

File: packages/highlight-pointer-bin/highlight-pointer-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+name="highlight-pointer-bin"
+pkgname="highlight-pointer"
+gives="highlight-pointer"
+version="1.1.3"
+
```
Comment (Henryws) [inline packages/highlight-pointer-bin/highlight-pointer-bin.pacscript:7]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/highlight-pointer-bin/highlight-pointer-bin.pacscript (hunk 1 above)
Comment (0x262D) [reply packages/highlight-pointer-bin/highlight-pointer-bin.pacscript]: Fixed

File: packages/highlight-pointer-bin/highlight-pointer-bin.pacscript (hunk 2 above)
Comment (0x262D) [reply packages/highlight-pointer-bin/highlight-pointer-bin.pacscript]: Fixed

**Review by Henryws — APPROVED** [review]

**Pull request merged**